src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private static final int numRandCol = Color.values().length;
+
     public String getRandomColor() {
-        return null;
+        int index = new Random().nextInt(numRandCol);
```

Comment:
create a global Random field and use it here, so you won't need to create an instance of it each time you call this method